packages/amplify-migration-template-gen/src/template-generator.ts

```diff
@@ -124,27 +125,27 @@ class TemplateGenerator {
       });
       await migrationReadMeGenerator.initialize();
 
-      const {
-        oldTemplate: oldGen1Template,
-        newTemplate: newGen1Template,
-        parameters: gen1StackParameters,
-      } = await categoryTemplateGenerator.generateGen1PreProcessTemplate();
+      const { newTemplate: newGen1Template, parameters: gen1StackParameters } =
+        await categoryTemplateGenerator.generateGen1PreProcessTemplate();
       assert(gen1StackParameters);
-      await migrationReadMeGenerator.renderStep1(oldGen1Template, newGen1Template, gen1StackParameters);
+      console.log(`Updating Gen1 ${category} stack...`);
+      const gen1StackUpdateStatus = await tryUpdateStack(this.cfnClient, gen1CategoryStackId, gen1StackParameters, newGen1Template);
+      assert(gen1StackUpdateStatus === CFNStackStatus.UPDATE_COMPLETE);
```

Comment:
What happens if the stack status is not `UPDATE_COMPLETE`? Do customers retry?

Reply:
Yeah it would fail at this point and they can re run the command.